packages/commonwealth/client/scripts/views/pages/Community/Groups/common/helpers/index.ts

```diff
@@ -26,7 +26,6 @@ export const makeGroupDataBaseAPIPayload = (
     if (
       x.requirementType === SPECIFICATIONS.ERC_20 ||
       x.requirementType === SPECIFICATIONS.ERC_721 ||
-      x.requirementType === SPECIFICATIONS.ERC_1155 ||
```

Comment:
Either we need to merge master or some conflicts were incorrectly resolved but this ERC_1155 code should not be removed.